neopixel.py

```diff
@@ -142,11 +144,14 @@ def _set_item(self, index, value):
                 r = 0
                 g = 0
                 b = 0
-        elif len(value) == self.bpp:
-            if self.bpp == 3:
-                r, g, b = value
+        elif (len(value) == self.bpp) or ((len(value) == 3) and (self.bpp == 4)):
+            if len(value) == 3:
+                r, g, b = value[0:3]
             else:
                 r, g, b, w = value
```

Comment:
This if/else could be replaced with a ternary:
```python
r, g, b, w = value if len(value)==4 else value+(0,)
```

Reply:
Works for me! Thanks.